.travis.yml

```diff
@@ -1,16 +1,28 @@
 language: erlang
 otp_release:
-  - 19.3
+- 19.3
 install:
-  - make
-  - ./rebar3 update
+- make
+- "./rebar3 update"
 script:
-  - make test
-  - make xref
-  - make dialyzer
-  - make lint
+- ./rebar3 ct --suite test/plumtree_SUITE --group hyparview --case broadcast_high_client_test
+# - make xref
```

Comment:
So, should I be reviewing this for a merge or should I wait until the ```!drop``` commits are removed?